tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -125,7 +125,7 @@ def test_local_secret_not_tensor(get_clients) -> None:
     assert np.allclose(torch.tensor(x_float), result)
 
 
-@pytest.mark.parametrize("nr_clients", [2, 3, 4, 5])
+@pytest.mark.parametrize("nr_clients", [2, 5])
```

Comment:
Agreed with @gmuraru, why do we change this?

Reply:
I have addressed this as a general comment. Let me know what you think.

Reply:
Let's also add the "3" value for those -- I think that 2, 3, 5 should suffice - 2 for FSS, 3 for Falcon, and 5 for a more general case.